server/main.lua

```diff
@@ -4,7 +4,14 @@ TriggerEvent('esx:getSharedObject', function(obj) ESX = obj end)
 
 RegisterNetEvent('blarglebus:finishRoute')
 AddEventHandler('blarglebus:finishRoute', function(amount)
+    local xPlayer = ESX.GetPlayerFromId(source)
+    
+    if xPlayer.job.name == 'busdriver' then
     ESX.GetPlayerFromId(source).addMoney(amount)
+    else
+        print(('blarglebus: %s attempted to exploit the job!'):format(xPlayer.identifier))
+	xPlayer.kick("You are kicked because of cheat!")
```

Comment:
Could you update these to use ESX's locale system, so the messages will be translated?

Something like...
```
        print(_U('exploit_attempted_log_message', xPlayer.identifier))
        xPlayer.kick(_U('exploit_attempted_kick_message'))
```

And then move the text of the message to locales/en.lua.